test/image.bats

```diff
@@ -45,7 +45,8 @@ function teardown() {
 	start_crio
 
 	pod_id=$(crictl runp "$TESTDATA"/sandbox_config.json)
-	sed -e "s/%VALUE%/quay.io\/crio\/redis:alpine/g" "$TESTDATA"/container_config_by_imageid.json > "$TESTDIR"/ctr_by_imagetag.json
+
+	jq '.image.image = "quay.io/crio/redis:alpine"' "$TESTDATA"/container_config_by_imageid.json > "$TESTDIR"/ctr_by_imagetag.json
```

Comment:
This is weird. The first commit in this PR removes this whitespace, the second one adds it back :-\